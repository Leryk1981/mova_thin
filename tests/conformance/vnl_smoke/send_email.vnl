@timeout_ms=12000
@retry.max_attempts=2
@retry.backoff_ms=400
emit channel="smtp" payload='{"to":"user@example.com","subject":"Hello","text":"Hi"}'

