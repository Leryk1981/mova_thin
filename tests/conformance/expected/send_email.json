{
  "mova_version": "3.3",
  "id": "send_email",
  "description": "Generated from VNL",
  "policies": {
    "timeout_ms": 12000,
    "retry": {
      "backoff_ms": 400
    }
  },
  "actions": [
    {
      "type": "emit_event",
      "channel": "smtp",
      "payload": {
        "to": "user@example.com",
        "subject": "Hello",
        "text": "Hi"
      }
    }
  ]
}